airbyte-persistence/job-persistence/build.gradle

```diff
@@ -0,0 +1,35 @@
+plugins {
+    id "java-library"
+}
+
+dependencies {
+    implementation project(':airbyte-commons')
+    implementation project(':airbyte-commons-docker')
```

Comment:
That might be part of the current dependency mess but I don't think we should have this.

Reply:
Agreed.  This is here because the `DefaultSyncJobFactory` currently calls `DockerUtils.getTaggedImageName` and this PR keeps existing functionality as it.  A follow-up PR will start cleaning up a lot of the current dependency mess.